ui/webui/Makefile.am

```diff
@@ -63,6 +63,7 @@ install-data-hook: $(WEBPACK_TEST)
 	mkdir -p $(DESTDIR)/usr/share/metainfo/
 	cp org.cockpit-project.$(PACKAGE_NAME).metainfo.xml $(DESTDIR)/usr/share/metainfo/
 	cp webui-desktop $(DESTDIR)/usr/libexec/
+	ln -sTfr $(DESTDIR)/usr/share/pixmaps/fedora-logo-sprite.svg $(DESTDIR)/usr/share/cockpit/$(PACKAGE_NAME)/logo.svg
```

Comment:
I don't think this is a good idea - this way we will essentially ship a symlink, that will not point to anything on the boot.iso and possibly elsewhere. IMHO, its should be simpler and more robust to just bundle the icon for now instead, until the Lorax template issues is resolved.

Reply:
The only problem I see with bundling the icon is that it will be hardcoded and thus we will need to manually adjust with possible logo adjustments that will happen with future fedora versions.

Regarding the boot.iso I don't know why you care - as this won't be a delivered product now.

Reply:
It's the tested thing though, even if not delivered?

Reply:
It's now delivered also for the boot.iso, so no broken symlinks.